sql/src/test/java/io/crate/operation/projectors/ProjectionToProjectorVisitorTest.java

```diff
@@ -80,6 +81,7 @@ public void prepare() {
         threadPool = new TestThreadPool("testing");
         visitor = new ProjectionToProjectorVisitor(
             mock(ClusterService.class),
+            mock(NodeJobsTracker.class),
```

Comment:
since the NodeJobsTracker has no dependencies we could just create a real instance